src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -67,6 +72,8 @@ def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_w
                 administrator_login_password=administrator_login_password,
                 version=version,
                 ssl_enforcement=ssl_enforcement,
+                minimal_tls_version=minimal_tls_version,
```

Comment:
MariaDB does not support minimal_tls_version. can remove this

Reply:
I am removing it in the params. So this should be fine. If the user tried to provide this value it will show unrecognized arguments. Going ahead we will be supporting TLS for mariadb as well and then its one line change